apps/api-extractor/src/analyzer/ExportAnalyzer.ts

```diff
@@ -863,9 +872,18 @@ export class ExportAnalyzer {
     exportSymbol: ts.Symbol
   ): AstModule {
     const moduleSpecifier: string = this._getModuleSpecifier(importOrExportDeclaration);
+    const mode =
+      importOrExportDeclaration.moduleSpecifier &&
+      ts.isStringLiteralLike(importOrExportDeclaration.moduleSpecifier)
+        ? TypeScriptInternals.getModeForUsageLocation(
+            importOrExportDeclaration.getSourceFile(),
+            importOrExportDeclaration.moduleSpecifier
+          )
+        : undefined;
```

Comment:
Ideally, you could leverage `_getModuleSpecifier` to return the *node* and get the string out of that. It's not critical, I wouldn't block this PR on it.

Reply:
Leaving this open to get feedback from maintainers here as well.